api/src/main/java/org/pmiops/workbench/exceptions/ExceptionAdvice.java

```diff
@@ -1,42 +1,55 @@
 package org.pmiops.workbench.exceptions;
 
+
+import com.ecwid.maleorang.MailchimpException;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import org.pmiops.workbench.model.ErrorResponse;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
-import org.springframework.web.bind.annotation.ResponseStatus;
 
 @ControllerAdvice
 public class ExceptionAdvice {
 
-  private static final Logger log = Logger.getLogger(ExceptionAdvice.class.getName());
-
   @ExceptionHandler({Exception.class})
   public ResponseEntity<?> serverError(Exception e) {
-    int statusCode = 500;
-    if (e.getClass().getPackage().getName().equals(
-        ExceptionAdvice.class.getPackage().getName())) {
-      ResponseStatus responseStatus = e.getClass().getAnnotation(ResponseStatus.class);
-      if (responseStatus != null) {
-        statusCode = responseStatus.value().value();
-        log.log(Level.WARNING, "[{0}] {1}: {2}",
-            new Object[]{statusCode, e.getClass().getSimpleName(), e.getMessage()});
-        if (statusCode < 500) {
-          if (e instanceof WorkbenchException) {
-            ErrorResponse errorResponse = ((WorkbenchException) e).getErrorResponse();
-            return ResponseEntity.status(statusCode).body(
-                ((WorkbenchException) e).getErrorResponse());
-          }
-          return ResponseEntity.status(statusCode).body(ExceptionUtils.errorResponse(
-              e.getMessage()));
-        }
+    ErrorResponse errorResponse = new ErrorResponse();
+    Integer statusCode = HttpStatus.INTERNAL_SERVER_ERROR.value();
+
+    // if this error was thrown by another error, get the info from that exception
+    Throwable cause = e;
```

Comment:
Since this is sometimes the "cause" and sometimes the original error that triggered this method, I might name this variable something like, "relevantError."